VATSpy.dat

```diff
@@ -9728,12 +9731,14 @@ NZWB|Woodbourne (Mil/Civ)|-41.518333|173.870278||NZCT|0
 NZWD|Williams Field|-77.86667|166.96639||NZCM|0
 NZWE|Kauaroa Bay|-36.817972|175.065556||NZAA|0
 NZWF|Wanaka|-44.738718|169.237232||NZCS|0
+NZWG|Whangamata|-37.200556|175.866667||NZAA|0
+NZWH|Wellington Hospital|-41.308333|174.779444||NZCT|0
 NZWJ|Wellsford|-36.284694|174.522778||NZAA|0
 NZWK|Whakatane|-37.922546|176.920215|WHK|NZCB|0
 NZWL|West Melton|-43.474615|172.388591||NZCS|0
 NZWM|Waimate|-44.789092|171.075519||NZCS|0
-NZWN|Wellington|-41.335264|174.803487|WLG|NZCT|0
 NZWN|Wellington|-41.335264|174.803487|NZWN-F|NZCT|1
+NZWN|Wellington|-41.335264|174.803487|WLG|NZCT|0

```

Comment:
By the look of it, WLG is not used as a logon for this airport, atleast I can not find it being used. However NZWN_DEL/GND/TWR/APP is being used.

If this is correct, then we should only have one entry for the airport, being `NZWN|Wellington|-41.335264|174.803487|NZWN-F|NZCT|0`